styles.css

```diff
@@ -30,7 +48,266 @@
 /* ------------------------------- 4. Areas ---------------------------------------- */
 
 /* --------------- 4.1 Header ------------------------- */
+.navbar-outer-container {
+    display: flex;
+    background: rgb(72,69,228);
+    background: linear-gradient(168deg, rgba(72,69,228,1) 0%, rgba(87,34,205,1) 35%);
+}
+
+
+.navbar-items-inner-container {
+    background-color: #5722CD;
+    font-family: "Roboto", sans-serif;
+    text-decoration-color: white;
+    display: flex;
+    padding: 25px;
+    justify-content: end;
+    list-style: none;
+    gap: 100px;
+    color: white;
+
+}
+
+.navbar-items-inner-container a:link {
+    color: white;
+}
+.navbar-items-inner-container a:visited {
+    color: white;
+}
 
 /* --------------- 4.2 Introduction ------------------- */
+.intro-outer-container-container {
+    display: flex;
+    flex-direction: column;
+    background-color: #5722CD;
+    margin-bottom: 25px;
+}
+
+.intro-h1-inner-container {
+    display: flex;
+
+    text-align: center;
+    color: white;
+    margin-top: 100px;
+    margin-right: 400px;
+    margin-left: 400px;
+    margin-bottom: 25px;
+    font-size: 40px;
+}
+
+.intro-inner-container {
+    display: flex;
+    flex-direction: column;
+    align-content: center;
+    text-align: center;
+    color: white;
+    font-family: "Roboto", sans-serif;
+    margin-right: 400px;
+    margin-left: 400px;
+    margin-bottom: 25px;
+    font-size: 15px;
+}
+
+.button {
+    display: flex;
+    flex-basis: auto;
+    align-content: center;
+    justify-content: center;
+    background: #ED4385;
+    font-family: "Roboto", sans-serif;
+    color: white;
+    font-size: 10px;
+    text-transform: uppercase;
+    padding: 10px 10px;
+    border-radius: 25px;
+    margin-right: 690px;
+    margin-left: 690px;
+    margin-bottom: 100px;
+    text-decoration: none;
+    box-shadow: 0 4px 6px rgba(0, 0, 0, 0.2);
+
+
+}
+
+.img-container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+}
+
+.intro-2-outer-container{
+    display: flex;
+    flex-direction: column;
+    padding: 50px;
+
+}
+
+.text-1 {
+    text-align: center;
+    font-family: "Merriweather", serif;
+    color: #4845E4;
+    font-size: 25px;
+    font-weight: bold;
+    padding: 15px 20px;
+    margin-right: 400px;
+    margin-left: 400px;
+
+
+}
+
+.text-2 {
+    text-align: center;
+    font-family: "Merriweather", serif;
+    color: #4845E4;
+    font-size: 25px;
+    font-weight: bold;
+    padding: 15px 20px;
+    margin-right: 400px;
+    margin-left: 400px;
+    font-weight: bold;
+
```

Comment:
Ik weet dat het verleidelijk is om Enters in te voegen, maar dat is over het algemeen niet toegestaan (behalve ééntje tussen het begin en einde van een nieuwe selector)